docs-ref-conceptual/node-azure-tools.md

```diff
@@ -13,16 +13,16 @@ ms.author: tarcher
 ---
 
 # Azure tools for Node.js developers
-Whatever your operating system, we have a great set of recommended tools to help you develop using Node.js on Azure.
+We recommend these great tools to develop using Node.js on Azure, whatever your operating system.
```

Comment:
Proposed...

We recommend these great tools to develop using Node.js on Azure, no matter what your operating system is.